includes/class-wc-calypso-bridge-setup.php

```diff
@@ -35,16 +35,32 @@ public static function get_instance() {
 	 * Constructor.
 	 */
 	private function __construct() {
+		add_action( 'admin_init', array( $this, 'redirect_store_details_onboarding' ) );
```

Comment:
Rather than the generic `admin_init`, is there [a more specific `load-$page_hook` action](https://github.com/WordPress/wordpress-develop/blob/trunk/src/wp-admin/admin.php#L217-L237) that could be hooked into instead?

Reply:
Great idea! The `load-woocommerce_page_wc-settings` hooks seem perfect for this job.